tests/Feature/EventsPageTest.php

```
@@ -0,0 +1,69 @@
+    {
+        $response = $this->get('/events');
+
+        $response->assertRedirect('/login');
+    }
+
+    public function test_events_page_shows_events_from_today(): void
+    {
+        $user = User::factory()->create();
+        
+        // Create an event from today
+        $todayEvent = Event::factory()->create([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$todayEvent'. (Unused Code Rules)

(UnusedLocalVariable)